src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```
@@ -2,6 +2,7 @@
 @import "../basic.css";
 @import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@600&display=swap');
 @import "../tabs.css";
+@import "../sg_gallery.css";
```

```suggestion
@import "../sg_gallery.css";
```

From this line, I'm not sure if a file called `sg_gallery.css` should be in the ``_static`` folder or not. I am no expert in CSS or HTML, so it is your call.

---

Yes if we are using sphinx_gallery and importing the package, then it will be there in _static folder.